apply/current_cluster.go

```diff
@@ -96,6 +96,9 @@ func deleteNode(name string) error {
 func DeleteNodes(nodeIPs []string) error {
 	logger.Info("delete nodes %s", nodeIPs)
 	nodes := listNodes()
+	if nodes == nil {
+		return fmt.Errorf("failed to get node list")
```

Comment:
failed to get node list: no nodes found

We could show error message as explicit as possible.